toonz/sources/toonz/cellselection.cpp

```diff
@@ -1519,6 +1543,13 @@ void TCellSelection::pasteCells() {
       DVGui::error(QObject::tr("No data to paste."));
       return;
     }
+
+    if (viewer && !viewer->orientation()->isVerticalTimeline()) {
+      int cAdj = cellData->getColCount() - 1;
+      c0 -= cAdj;
+      c1 -= cAdj;
+    }
```

Comment:
(It is not an urgent problem, but would be an issue for the future.)
You are adjusting the paste destination in order to make the cells to be pasted below the selected cell.
IMO it would better to apply the same adjustment to `Paste Into` and `Paste Numbers` commands as well, in order to gain consistency.

Reply:
I like consistency. :)  Updated to make sure all the paste options apply the timeline adjustment.